src/core/annotation.js

```diff
@@ -4587,18 +4590,19 @@ class InkAnnotation extends MarkupAnnotation {
       `${numberToString(outline[4])} ${numberToString(outline[5])} m`
     );
     for (let i = 6, ii = outline.length; i < ii; i += 6) {
-      if (isNaN(outline[i]) || outline[i] === null) {
+      if (isNaN(outline[i])) {
         appearanceBuffer.push(
           `${numberToString(outline[i + 4])} ${numberToString(
             outline[i + 5]
           )} l`
         );
       } else {
-        const curve = outline
-          .slice(i, i + 6)
-          .map(numberToString)
-          .join(" ");
-        appearanceBuffer.push(`${curve} c`);
+        const [c1x, c1y, c2x, c2y, x, y] = outline.slice(i, i + 6);
+        appearanceBuffer.push(
+          `${numberToString(c1x)} ${numberToString(c1y)} ` +
+            `${numberToString(c2x)} ${numberToString(c2y)} ` +
+            `${numberToString(x)} ${numberToString(y)} c`
```

Comment:
*Same question as above.*
```suggestion
          [c1x, c1y, c2x, c2y, x, y].map(numberToString).join(" ") + " c"
```